.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6
+    working_directory:  ~/fiat-test
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies  # Install with sudo as tests not run as superuser in circleci/python
+          command: sudo pip install flake8 pytest pydocstyle numpy sympy coverage coveralls --upgrade
+      - run:
+          name: Install FIAT
+          command: pip install . --user
+      - run:
+          name: Run flake8 and pydocstyle tests
+          command: |
+              python -m flake8 .
+              python -m pydocstyle .
+      - run:
+          name: Run unit tests
+          command: |
+              DATA_REPO_GIT="" coverage run --source FIAT -m pytest -v test/
+              if [ -v COVERALLS_REPO_TOKEN ]; then
+                coveralls
+              fi
```

Comment:
This looks like a merge/rebase bork.  Do you really intend to configure Circle CI?

Reply:
Must have been.  They switched a while back.  Removed